app/models/service_update.rb

```diff
@@ -5,16 +5,40 @@ class ServiceUpdate
 
   include ActiveModel::Model
 
-  attr_accessor :date, :title, :content
+  attr_accessor :date, :title, :content, :summary
 
   def id
     title.parameterize
   end
 
   def self.all
-    YAML.load_file(SERVICE_UPDATES_YAML_FILE).map do |params|
+    updates = YAML.load_file(SERVICE_UPDATES_YAML_FILE).map do |params|
       new(params)
     end
+
+    updates.sort_by { |update| Date.parse(update.date) }.reverse
+  end
+
+  def self.find_by_id(id)
+    all.find { |update| update.id == id }
+  end
+
+  def summary_text
+    return summary if summary.present?
+
+    content_lines = content.split("\n").compact_blank
+    first_paragraph = content_lines.first&.strip
+
+    if first_paragraph && first_paragraph.length > 200
+      "#{first_paragraph[0, 200]}..."
+    else
+      first_paragraph || ""
+    end
+  end
```

Comment:
proposed "auto" summary - grab the first paragraph (up to 200 chars) and display that